pkg/ipmi/ipmi.go

```diff
@@ -173,3 +176,19 @@ func (i *IPMI) ShutoffWatchdog() error {
 
 	return nil
 }
+
+func (i *IPMI) AddSel(data [16]byte) error {
```

Comment:
Should a `SelEventRecord` type be defined here instead of using `[16]byte`?

https://github.com/ipmitool/ipmitool/blob/master/include/ipmitool/ipmi_sel.h#L127

Reply:
+1

Can we also rename this to be something like "LogSystemEvent"? I had to look up what SEL was.

Reply:
SEL is System Event Log (bing IPMI SEL). For brevity, can we  just define it in one place (e.g. pkg/ipmi/doc.go) and use SEL (we don't say Transmission Control Protocol or Intelligent Platform Management Interface everywhere either :-) in our function names? Not Sel, SEL :-)

Reply:
How about `LogEvent`?

Reply:
SEL is not as common an acronym to throw around as IPMI or TCP.

The set of people that know what SEL is compared to IPMI is probably fairly low.

Reply:
I  renamed the function to `AddSystemEventLog`.